### DISCUSSION TITLE
How to remove the character limit of a password when authen with user password

### DISCUSSION THREAD

**Discussion by AltAdamNguyen** [schwabe/ics-openvpn#1621] [Configuration problems] [answered]
I have a profile openvpn use auth by user and pass, when connecting profile status return is Auth_Failed, so I check in the server my password I entered is missing 10 characters.

**Comment by schwabe** [comment]
How long is your password? More than 256 characters?

**Reply by AltAdamNguyen** [reply]
When i check on the server only get 127 characters, the rest is cut off

**Reply by schwabe** [reply]
Yeah. But 127 character is the limit that is actually put in place by the OpenVPN source code. 

```
#define USER_PASS_LEN 128
```
Why do you need to more than 127 characters as password?

**Reply by AltAdamNguyen** [reply]
how can I edit?

**Reply by schwabe** [reply, marked as answer]
This is something that is backed into OpenVPN itself. Without changing OpenVPN itself it is not possible. And a password with more than 127 characters seem wrong anyway.

**Reply by AltAdamNguyen** [reply]
Oke. Thank you for your support
